coalib/bearlib/aspects/Formatting.py

```diff
@@ -0,0 +1,355 @@
+        (True, False), default=False)
+
+
+@Spacing.subaspect
+class BlankLine:
+    """
+    A line with zero characters.
+    """
+    class docs:
+        example = """
+        name = input('What is your name?')
+
+
+        print('Hi, {}'.format(name))
+        """
+        example_language = 'python'
+        importance_reason = """
+        Various programming styles use blank lines in different places.
+        The usage of blank lines affects the readability, maintainability and
+        length of a code i.e blank lines can either make code longer, less
+        readable and maintainable or do the reverse.
+        """
+        fix_suggestions = """
+        Following specific rules about the usage of blank lines: using them
+        only when necessary.
+        """
+
+
+@BlankLine.subaspect
+class BlankLineAfterDeclaration:
+    """
+    Those found after declarations.
+    """
+    class docs:
+        example = """
+        int main ()
+        {
+          int a;
+          return 0;
+        }
+        """
+        example_language = 'C'
+        importance_reason = """
+        Having a specific and reasonable number of blank lines after every
+        block of declarations improves on the readability of the code.
+        """
+        fix_suggestions = """
+        `BlankLintAfterDeclaration` issues can be fixed specifying (and of
+        course using) a reasonable number of blank lines to use after block
+        declaration.
+        """
+    blank_lines_after_declarations = Taste[int](
+        'Represents the number of blank lines after declarations',
+        (0, 1, 2), default=0)
+
+
+@BlankLine.subaspect
+class BlankLineAfterProcedure:
+    """
+    Those found after procedures or functions.
+    """
+    class docs:
+        example = """
+        void proc(void){
+            printf("this does nothing");
+        }
+
+        int add(float a, float b){
+            return a + b;
+        }
+        """
+        example_language = 'C++'
+        importance_reason = """
+        Having a specific and reasonable number of blank lines after every
+        procedures improves on the readability of the code.
+        """
+        fix_suggestions = """
+        `BlankLintAfterProcedure` issues can be fixed specifying (and of
+        course using) a reasonable number of blank lines to use after
+        procedures' definition.
+        """
+    blank_lines_after_procedures = Taste[int](
+        'Represents the number of blank lines to use after a procedure or a'
+        ' function', (0, 1, 2), default=1)
+
+
+@BlankLine.subaspect
+class BlankLineAfterClass:
+    """
+    Those found after classes' definitions.
+    """
+    class docs:
+        example = """
+        class SomeClass:
+            def __init__(self):
+                raise RuntimeError('Never instantiate this class')
+
+
+        def func():
+            pass
+        """
+        example_language = 'python'
```

Comment:
in Python 2, a class should inherit from `object`.  In Python 3, it shouldnt.  So this needs to be a version specific example.
Change this to a 'Python 3' example?

Also, why is it 'python' instead of 'Python'